app/ui/browser/views/browser.jsx

```diff
@@ -184,5 +184,6 @@ function attachIPCRendererListeners(browserView) {
 
   // @TODO Not yet implemented
   ipcRenderer.on('show-bookmarks', () => dispatch(ipcActions.showBookmarks()));
-  ipcRenderer.on('open-bookmark', bookmark => dispatch(ipcActions.openBookmark(bookmark)));
+  ipcRenderer.on('open-bookmark',
+    (_event, bookmark) => dispatch(ipcActions.openBookmark(bookmark)));
```

Comment:
So did this just not used to work?  Or did it not matter what was passed into openBookmark?